libs/pxweb2-ui/src/lib/components/Select/Select.tsx

```diff
@@ -0,0 +1,143 @@
+import cl from 'clsx';
+import classes from './Select.module.scss';
+import Label from '../Typography/Label/Label';
+import Button from '../Button/Button';
+import BodyShort from '../Typography/BodyShort/BodyShort';
+
+export type SelectOption = {
+  label: string;
+  value: string;
+};
+
+export interface SelectProps {
+  variant?: 'default' | 'inVariableBox';
+  label: string;
+  hideLabel?: boolean;
+  defaultOption?: string;
+  options: SelectOption[];
```

Comment:
We should probably add onChange here. The purpose of passing a onChange function to the select component is to send data out of the component when a new set of items is selected. A example of how this is done can be seen in the checkbox component 😄 

When selecting options in the modal component the data will then flow from the modal to this component, and then have to flow out of this component to the parent. onChange is how we solve this.

Reply:
Now I have added the onChange event